app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryDescription/ExpandedStoryDescriptionTextArea.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { MentionsInput, Mention } from 'react-mentions';
+import PropTypes from 'prop-types';
+
+const ExpandedStoryDescriptionTextArea = ({
+  onEdit,
+  disabled,
+  description,
+  users
+}) => {
+  const formatMention = (_, display) => `@${display}`;
+  const data = users.map(({ id, username }) => ({ id, display: username }));
```

Comment:
Can we improve the name for this variable? We could also extract the mapping function to a variable. Something like this:

```js
const mentionableUsers = users.map(extractMentionData);
```